index.html

```diff
@@ -456,32 +421,171 @@ <h2 id="constraints">
+        <h2 id="constrainable-properties">
+          Constrainable Properties for Captured Display Surfaces
         </h2>
-        <p>
-          Following new <a href="https://www.w3.org/TR/mediacapture-streams/#defining-a-new-constrainable-property">
-          Constrainable Properties</a> are defined that allow an application to observe the properties
-          of the selected <a>display surface</a>. Since the source of media cannot be changed after a
-          <code>MediaStreamTrack</code> has been returned and constraints do not affect the
-          selection of <a>display surfaces</a>, these constrainable properties cannot be changed by an
-          application.
-        </p>
-        <p>
-          The <code><a data-link-for="MediaTrackConstraintSet">displaySurface</a></code> constraint
-          allows an application to observe the type of <a>display surface</a> that is being
-          captured.
-        </p>
-        <p>
-          The <code><a data-link-for="MediaTrackConstraintSet">logicalSurface</a></code> constraint
-          allows an application to observe whether the surface that is captured is a <a>logical
-          display surface</a>, rather than the <a>visible display surface</a>.
-        </p>
-        <p>
-          The <code><a data-link-for="MediaTrackConstraintSet">cursor</a></code> constraint
-          allows an application to specify if the cursor should be included in the captured
-          display.
-        </p>
+        <p>Constraints serve a different purpose for display device capture. They
+        do not aid discovery, instead they are applied only after user-selection.
+        Some of these constraints enable user agent processing like downscaling
+        and frame decimation, as well as display-specific features. Others enable
+        observation of inherent properties of a user-selected
+        <a>display surface</a>, as capabilities and settings.</p>
+        <p>The following new and existing <code><a>MediaStreamTrack</a></code>
+        <a href="https://www.w3.org/TR/mediacapture-streams/#constrainable-properties">
+        Constrainable Properties</a> are defined to apply to the user-selected
+        video <a>display surface</a>, with the following behavior:</p>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Property Name</th>
+              <th>Type</th>
+              <th>Behavior</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="def-constraint-width">
+              <td><dfn>width</dfn></td>
+              <td><code><a>ConstrainULong</a></code></td>
+              <td>The width or width range, in pixels. As a capability, max MUST
+              reflect the <a>display surface</a>'s width, and min MUST reflect
+              the width of the smallest aspect-preserving representation
+              available through downscaling by the user agent.</td>
```

Comment:
As a setting: [MediaTrackSettings](https://w3c.github.io/mediacapture-main/getusermedia.html#dom-mediatracksettings) contains only values, not ranges, so this should be clear.

This is mirrored on [similar language in gUM](https://www.w3.org/TR/mediacapture-streams/#constrainable-properties). Describing the capability range is meant to be sufficient to deduce the other behaviors: only settings within that range are valid, and because the original size is the max, it follows that sizes larger than the original are not allowed. No up-scaling is mentioned specifically in a separate section.

As a capability: I broke out a section dedicated to detailing the downscaling behavior. Too much for the table.